packages/react-instantsearch-hooks-dom/src/ui/RefinementList.tsx

```diff
@@ -88,7 +92,13 @@ export function RefinementList({
   return (
     <div
       {...props}
-      className={cx('ais-RefinementList', classNames.root, className)}
+      className={cx(
+        'ais-RefinementList',
+        classNames.root,
+        items.length === 0 &&
```

Comment:
should this use `canRefine` instead? or is that different?

Reply:
Good idea, we can use it. I'll pass it from the widget.